portfolio/src/main/webapp/index.html

```diff
@@ -207,9 +207,9 @@ <h4>my relevant coursework</h4>
 
         <section id="comments">
           <h3>arrington's friends comments</h3>
-          <form action="/data?pag=0" method="POST">
+          <form id="dropdown" action="/data" method="POST">
```

Comment:
Since there is no submit on this form, can you remove the `action` and `method` attributes from the `form` tag.

Reply:
done!